packages/idos-sdk-js/src/lib/utils.ts

```diff
@@ -1,25 +1,28 @@
 import { decodeBase58, toBeHex } from "ethers";
-import * as nearAPI from "near-api-js";
+import { connect } from "near-api-js";
 
 export async function getNearFullAccessPublicKeys(
   namedAddress: string,
 ): Promise<string[] | undefined> {
-  const { connect } = nearAPI;
   const connectionConfig = {
     networkId: import.meta.env.VITE_IDOS_NEAR_DEFAULT_NETWORK,
     nodeUrl: import.meta.env.VITE_IDOS_NEAR_DEFAULT_RPC_URL,
   };
   const nearConnection = await connect(connectionConfig);
 
   try {
-    const response = await nearConnection.connection.provider.query({
+    // biome-ignore lint/suspicious/noExplicitAny: <explanation>
+    const response: any = await nearConnection.connection.provider.query({
       request_type: "view_access_key_list",
       finality: "final",
       account_id: namedAddress,
     });
     return response.keys
-      .filter((element: object) => element.access_key.permission === "FullAccess")
-      ?.map((i) => i.public_key);
+      .filter(
+        (element: Record<string, Record<string, string>>) =>
+          element.access_key.permission === "FullAccess",
+      )
+      ?.map((i: Record<string, string>) => i.public_key);
```

Comment:
These types aren't quite right. I think we want https://github.com/near/near-api-js/blob/ed54e104fdb277eb3c91fb350f51aea41068d7a5/packages/types/src/provider/response.ts#L120, maybe?